hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -517,27 +524,56 @@ def reranker_settings(reranker_type):
 }"""
 
         with gr.Row():
-            input_file = gr.File(
-                value=[os.path.join(resource_path, "demo", "test.txt")],
-                label="Docs (multi-files can be selected together)",
-                file_count="multiple",
-            )
-            input_schema = gr.Textbox(value=schema, label="Schema")
-            info_extract_template = gr.Textbox(value=SCHEMA_EXAMPLE_PROMPT, label="Info extract head")
             with gr.Column():
-                mode = gr.Radio(
-                    choices=["Test Mode", "Import Mode", "Clear and Import", "Rebuild Vector"],
-                    value="Test Mode",
-                    label="Build mode",
-                )
-                btn = gr.Button("Build Vector/Graph RAG")
+                with gr.Tab("file") as tab_upload_file:
```

Comment:
When I tried to switch between different tabs, I found that the files I uploaded to file would disappear after switching to text and then going back to file

![image](https://github.com/user-attachments/assets/081f3f54-498f-4d81-96ab-14a72199723a)

Reply:
It seems that Gradio Tab does not have an interface to retrieve which value to select, so it can only be left blank or modified to gradio Radio or Gradio Dropdown to solve it.

Reply:
> It seems that Gradio Tab does not have an interface to retrieve which value to select, so it can only be left blank or modified to gradio Radio or Gradio Dropdown to solve it.

Get it🫡